SCP/SCP.cpp

```diff
@@ -90,7 +90,7 @@ int main(int argument_count, char * argv[])
 
 	// TODO: Free all the memory we allocated
 	free_instance(&instance);
-	free(output_file_path);
+	//free(output_file_path);
```

Comment:
Its all good to leave it commented out. Also comment out the line where its generated though :)
All this was mainly for debugging anyway. But we'll need a more formal approach for that soon enough